### PR TITLE
[138] - Добавил TextField

### PR DESCRIPTION
**Pull request by KirBorisov96**

### CODE REVIEW

File: lib/src/widgets/views/textfields/textfield/textfield.dart [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+          crossAxisAlignment: CrossAxisAlignment.end,
+          children: <Widget>[
+            Flexible(
+              child: Stack(
+                children: <Widget>[
+                  Column(
+                    crossAxisAlignment: CrossAxisAlignment.end,
+                    children: <Widget>[
+                      SizedBox(
+                        height: (LayoutGrid.module * 7) / LayoutGrid.halfModule,
```
Comment (timbaton) [inline lib/src/widgets/views/textfields/textfield/textfield.dart:108]: Мне кажется можно просто взять doubleModule

File: lib/src/widgets/views/textfields/textfield/animatedLabel/textfield_animated_label.dart [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                      style: TextStyle(
+                        fontSize: labelFont.fontSize,
+                        color: labelTextColor,
+                        fontFamily: labelFont.fontFamily,
+                        fontWeight: labelFont.fontWeight,
+                      ),
+                    ),
+                  ),
+                  const Spacer(),
+                ],
```
[the remark this answers is not in the document]
Comment (KirBorisov96) [reply lib/src/widgets/views/textfields/textfield/animatedLabel/textfield_animated_label.dart:94]: да обязательно, надо чтобы вьюха к левому краю была прибита

File: example/lib/screens/standart_textfield_screen.dart [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+          ),
+        ),
+        centerTitle: true,
+        bottomOpacity: 0.0,
+        elevation: 0.0,
+        backgroundColor: colors.backgroundBasic.color(),
+      ),
+      body: Padding(
+        padding: const EdgeInsets.symmetric(
+          horizontal: 16,
```
Comment (Andyatkov) [inline example/lib/screens/standart_textfield_screen.dart:57]: Grid

**Review by Andyatkov — APPROVED** [review]

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 23
Version: 1671100086
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/138-textfield
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/23

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 24
Version: 1671100350
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/138-textfield
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/24

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 25
Version: 1671101399
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/138-textfield
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/25

**Review by Almazovivan — APPROVED** [review]

**Pull request merged**